crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1188,13 +1198,26 @@ impl Client {
     /// Convenience shorthand for [`create_room`][Self::create_room] with the
     /// given user being invited, the room marked `is_direct` and both the
     /// creator and invitee getting the default maximum power level.
-    pub async fn create_dm(&self, user_id: &UserId) -> Result<Room> {
-        self.create_room(assign!(create_room::v3::Request::new(), {
+    ///
+    /// # Arguments
+    ///
+    /// * `user_id` - The ID of the user to create a DM for.
+    /// * `encrypted` - Whether the DM should be encrypted.
+    pub async fn create_dm(&self, user_id: &UserId, encrypted: Encrypted) -> Result<Room> {
```

Comment:
Given that it's a shortcut, I'd enable encryption by default. Without adding a new arg.

Notice that typically clients do some more sophisticated check to decide for the default value of encryption, but this would be part of another task/story.
For sake of simplicity just enable encryption